README.md

```diff
@@ -1,236 +1,261 @@
+  - **color**: The color of the text in hexadecimal format (for example: color=FFFFFF)
+  - **font**: The font family to be used for the text (Bitmap Font file, there are some samples on resources folder, only is needed to use the Font name before the _ simbol. For example: Matrix or BTTF)
+  - **bordercolor**: The color of the text border in hexadecimal format (for example: color=FFAAFF)
+  - **bordersize**: The size of the text border (0 or 1)
+  - **cleanbg**: Clean the active screen (when not cleaned the text will showed over the current image/animation
+  - **animationin**: Display animation: `FadeIn|FadeOut|ScrollOffRight|ScrollOffLeft|ScrollOnLeft|ScrollOnRight|ScrollOffUp|ScrollOffDown|ScrollOnUp|ScrollOnDown|FillFadeIn|FillFadeOut|None`
+  - **animationout**: Display animation: `FadeIn|FadeOut|ScrollOffRight|ScrollOffLeft|ScrollOnLeft|ScrollOnRight|ScrollOffUp|ScrollOffDown|ScrollOnUp|ScrollOnDown|FillFadeIn|FillFadeOut|None`
+  - **duration**: time to present the scene in the DMD (If an animation is selected, the screen will remain black once the animation ends if the time is longer than the animation itself. If the time is -1, it will be permanent)
+
+## Use in DOFLinx
+
+To generate effects, DOFLinx uses `.MAME` files located in DOFLinx's MAME
+folder. DOFLinx can communicate with DOF2DMD, using DOFLinx `FF_DMD` command.
+The `FF_DMD` command can call any of the DOF2DMD APIs.
+
+### `DOFLinx.ini` file
+
+Here is a minimal DOFLinx.ini file which will work with `DOF2DMD`:
+
+```ini
+# location of your files and systems
+COLOUR_FILE=<DOFLinx path>\config\colours.ini
+DIRECTOUTPUTGLOBAL=<DOFLinx path>\config\GlobalConfig_b2sserver.xml
+PATH_MAME=<DOFLinx path>\MAME\
+MAME_FOLDER=<MAME executable path (note: it must be DOFLinx modified MAME version)>
+
+# When to activate, and more specifically what is the MAME process to kick things off
+PROCESSES=Mame64
+MAME_PROCESS=Mame64
+
+# DOF2DMD
+PATH_DOF2DMD=<location of DOF2DMD executable and settings.ini>
+```
+
+Note:
+
+- `PATH_DOF2DMD`: the location of DOF2DMD executable and settings.ini
+- `MAME_FOLDER`: MAME executable path which must be DOFLinx's [modified version of MAME](https://drive.google.com/drive/folders/1AjJ8EQo3AkmG2mw7w0fLzF9HcOjFoUZH)
+
+### Embedded commands
+
+DOFLinx will generate the following commands automatically:
+
+- When starting DOFLinx:
+  - `http://<host:port>/v1/version` - to check that DOF2DMD is up. DOFLinx will attempt to start it otherwise.
+  - `http://<host:port>/v1/display/picture?path=mame/DOFLinx` - to display the DOFLinx welcome picture
+- When starting a game:
+  - `http://<host:port>/v1/display/picture?path=mame/<rom-name>&duration=<duration>&animation=<animation>` - to display a PNG for the marquee
+- When playing a game:
+  - `http://<host:port>/v1/display/score?player=<active player>&score=<score>&cleanbg=<true|false>` - to display score of the given player

```

Comment:
Same as above. In its current state, DOFLinx only sends `http://<host:port>/v1/display/score?player=<active player>&score=<score>` when sending score.